### PR TITLE
[tw] Add content/zh-tw/observability.md

### LINKED ISSUE #2309: [tw] Localize `Observability` for Traditional Chinese
1. Localize observability for Traditional Chinese
2. https://glossary.cncf.io/observability/

### PR DESCRIPTION
**Pull request by hwchiu**
### Describe your changes
1. Origin content from https://glossary.cncf.io/observability/
2. 參考 [微軟官方翻譯](https://learn.microsoft.com/zh-tw/azure/site-reliability-engineering/articles/code??WT.mc_id=AZ-MVP-5003331) 將 Toil 翻譯為 繁瑣作業

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2309 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/observability.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: 可觀測性
+status: Completed
+category: 概念
+tags: ["屬性", "", ""]
+---
+
+可觀察性是一個系統屬性，其定義了系統能夠產生可執行的見解的程度。
+它允許使用者從這些外部輸出去理解系統狀態並採取行動來修正。
+
+計算機系統通過觀察低級信號進行衡量，例如 CPU 時間、記憶體、磁碟空間，以及更高級和業務信號，包括 API 回應時間、錯誤、每秒交易次數等。
```
Comment (pichuang) [inline content/zh-tw/observability.md:11]:
```suggestion
計算機系統通過觀察低階信號進行衡量，例如 CPU 時間、記憶體、磁碟空間，以及更高級和業務信號，包括 API 回應時間、錯誤、每秒交易次數等。
```

File: content/zh-tw/observability.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: 可觀測性
+status: Completed
+category: 概念
+tags: ["屬性", "", ""]
+---
+
+可觀察性是一個系統屬性，其定義了系統能夠產生可執行的見解的程度。
```
Comment (pichuang) [inline content/zh-tw/observability.md:8]:
```suggestion
可觀察性是一個系統屬性，其定義了系統能夠產生可執行見解的程度。
```

File: content/zh-tw/observability.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+它允許使用者從這些外部輸出去理解系統狀態並採取行動來修正。
+
+計算機系統通過觀察低級信號進行衡量，例如 CPU 時間、記憶體、磁碟空間，以及更高級和業務信號，包括 API 回應時間、錯誤、每秒交易次數等。
+這些可觀察的系統通過專業工具，即所謂的可觀察性工具，進行觀察（或監控）。這些工具的列表可以在 Cloud Native Landscape 的可觀察性部分 中查看。
```
Comment (pichuang) [inline content/zh-tw/observability.md:12]:
```suggestion
這些可觀察的系統通過專業工具，即所謂的可觀察性工具，進行觀察（或監控）。這些工具的列表可以在 [Cloud Native Landscape](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category) 的可觀察性部分 中查看。
```

File: content/zh-tw/observability.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+計算機系統透過觀察底層訊號來進行量測，例如 CPU 時間、記憶體用量，硬碟空間以及更上層和業務訊號，包含了 API 響應時間、錯誤、每秒交易次數等。
+這些可觀察的系統可被 [Cloud Native Landscape](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category) 列表中所列舉的可觀測性工具**觀測**(或監控)。
+
+可觀察的系統向操作人員提供有意義且可執行的資料，使他們能夠達到更有利的結果 (更快的意外響應，開發者生產力提升)並減少繁瑣作業與停機時間。
```
Comment (pichuang) [inline content/zh-tw/observability.md:17]:
```suggestion
可觀察的系統向操作人員提供有意義且可執行的資料，使他們能夠達到更有利的結果 (更快的事件應變，開發者生產力提升)並減少繁瑣作業與停機時間。
```

File: content/zh-tw/observability.md (hunk 1 above)
Comment (ydFu) [reply content/zh-tw/observability.md:11]:
```suggestion
計算機系統通過觀察低階信號（例如 CPU 時間、記憶體、磁碟空間）以及更高階和業務信號（包括 API 回應時間、錯誤、每秒交易次數等）進行衡量。
```

File: content/zh-tw/observability.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+可觀察性是一個系統屬性，其定義了系統能夠產生可執行的見解的程度。
+它允許使用者從這些外部輸出去理解系統狀態並採取行動來修正。
+
+計算機系統通過觀察低級信號進行衡量，例如 CPU 時間、記憶體、磁碟空間，以及更高級和業務信號，包括 API 回應時間、錯誤、每秒交易次數等。
+這些可觀察的系統通過專業工具，即所謂的可觀察性工具，進行觀察（或監控）。這些工具的列表可以在 Cloud Native Landscape 的可觀察性部分 中查看。
+
+計算機系統透過觀察底層訊號來進行量測，例如 CPU 時間、記憶體用量，硬碟空間以及更上層和業務訊號，包含了 API 響應時間、錯誤、每秒交易次數等。
+這些可觀察的系統可被 [Cloud Native Landscape](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category) 列表中所列舉的可觀測性工具**觀測**(或監控)。
```
Comment (ydFu) [inline content/zh-tw/observability.md:15]: This section repeats the translation.

File: content/zh-tw/observability.md (hunk 3 above)
Comment (ydFu) [reply content/zh-tw/observability.md:12]:
```suggestion
這些可觀察的系統通過專業工具，即所謂的可觀察性工具，進行**觀察**（或監控）。
這些工具的列表可以在 [Cloud Native Landscape 的可觀察性部分](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category) 中查看。
```

EN
```
These observable systems are **observed** (or monitored) through specialized tools, so-called observability tools. A list of these tools can be viewed in the [Cloud Native Landscape's observability section](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category).
```

Hyperlinks must be consistent with the original text

File: content/zh-tw/observability.md (hunk 5 above)
Comment (ydFu) [inline content/zh-tw/observability.md:15]: Repeat translation of this chapter

File: content/zh-tw/observability.md (hunk 1 above)
Comment (hwchiu) [reply content/zh-tw/observability.md]: Fixed

File: content/zh-tw/observability.md (hunk 2 above)
Comment (hwchiu) [reply content/zh-tw/observability.md]: Fixed

File: content/zh-tw/observability.md (hunk 3 above)
Comment (hwchiu) [reply content/zh-tw/observability.md]: Fixed, I accidentally kept my original draft, so what's why I have the repeat content before.

File: content/zh-tw/observability.md (hunk 4 above)
Comment (hwchiu) [reply content/zh-tw/observability.md]: Fixed

File: content/zh-tw/observability.md (hunk 5 above)
Comment (hwchiu) [reply content/zh-tw/observability.md]: Fixed

**Review by ydFu — APPROVED** [review]
LGTM,thank @hwchiu

**Review by pichuang — APPROVED** [review]
lgtm

**Pull request merged**